test/controller.go

```diff
@@ -323,3 +325,55 @@ var _ ktesting.Reactor = (*ResourceVersionReactor)(nil)
 func PrependResourceVersionReactor(f *ktesting.Fake) {
 	f.ReactionChain = append([]ktesting.Reactor{&ResourceVersionReactor{}}, f.ReactionChain...)
 }
+
+// EnsureConfigurationConfigMapsExist makes sure all the configmaps exists.
+func EnsureConfigurationConfigMapsExist(d *Data) {
```

Comment:
Thanks Yongxuan, shall we also refactor for the following file https://github.com/tektoncd/pipeline/blob/99226de6ef1c3f14e38e9b8cbc1fe0e60753a079/pkg/resolution/resolver/framework/testing/fakecontroller.go/#L123?

Reply:
No - that's specifically for the resolver config maps in the `tekton-pipelines-resolvers` namespace.